BUILD.md

```diff
@@ -59,11 +59,11 @@ list some basic and more used commands as example.
     ```sh
     ./waf configure --board bebop --static # Bebop or Bebop2
     ./waf configure --board edge           # emlid edge
-    ./waf configure --board fmuv3          # Pixhawk2/Cube using ChibiOS
+    ./waf configure --board fmuv3          # Pixhawk 1 & 3DR Pixhawk 2 boards
```

Comment:
This shouldn't mention PixHawk1.

Users should be choosing one of `PixHawk1` or `Pixhawk1-1M` - if they choose fmuv3 those with 1MB boards won't have much joy.

I'd be tempted to get rid of fmuv3 here altogether, but @proficnc seems to think keeping it is warranted.